src/daemon/emitter.cc

```diff
@@ -389,6 +448,16 @@ void Emitter::DoRemoteExecute(const Atomic<bool>& is_shutting_down,
       continue;
     }
 
+    if (incoming->has_flags() && incoming->flags().has_sanitize_blacklist()) {
+      Immutable content;
+      auto blacklist = incoming->flags().sanitize_blacklist();
+      std::string sanitize_blacklist_file_path = incoming->current_dir();
```

Comment:
This code duplicates with the one from `GetAdditionalSourceFiles`, maybe it's better to write a function for it (this could also help moving to a platform where `/` is not a path separator)?